puan/logic/plog/__init__.py

```diff
@@ -558,25 +558,37 @@ def evaluate(self, interpretation: typing.List[puan.SolutionVariable]) -> bool:
                 ),
             )
         )
+        return self.evaluate_propositions(interpretation_map)[self.id] > 0
```

Comment:
```suggestion
        return self.evaluate_propositions(interpretation_map)[self.id] >= 0
```

Reply:
No should be strict here. A zero = false, and 1 = True